docs/src/docs/asciidoc/user/getting-started.adoc

```diff
@@ -162,6 +162,8 @@ You can create an XML file to configure a `CacheManager`.
 ----
 include::{sourcedir310}/ehcache-xml/src/test/resources/configs/docs/getting-started.xml[tags=gettingStarted]
 ----
+NOTE: The use of memory units (such as 'B', 'kB' or 'MB') for heap resources is deprecated and will be removed in a future version.
```

Comment:
I wonder if we should add "(This only applies to heap tiers)" ... I know the language says "for heap resources", but since there a multiple things in the examples, and readers may be new to the concepts, they might think too broadly the the feature in full is deprecated (for all tiers) - especially since the example does not show using for the heap tier!

Reply:
I got your point, Agreed as well, and Updated my note to 
`NOTE: The use of memory units (such as 'B', 'kB' or 'MB') for heap resources is deprecated and will be removed in a future version.This only applies to heap tiers.`